.checkpatch.conf

```diff
@@ -73,3 +73,19 @@
 # BRACES reports this kind of messages:
 # braces {} are not necessary for any arm of this statement
 --ignore BRACES
+
```

Comment:
I think you should have all the new ignored rules in the same patch. Or, at the very least, have one patch for ``CHECKPATCH_OPTS`` and one for the other options.

Reply:
For me the `--ignore BRACES` is to correct an issue that came when MISRA was adopted, that's why I put that in a separate patch.
I can split the second patch in two, one for the `CHECKPATCH_OPTS`, another one for the checks to ignore when using `--strict` option to checkpatch.

Reply:
I've pushed a new version in that way (3 patches)